modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1096,8 +1097,9 @@ public final class IgniteSystemProperties {
      * <p>
      * NOTICE: Should be the default in Apache Ignite 3.0
      */
-    @SystemProperty("Enables fields to be written by BinaryMarshaller in sorted order. " +
-        "By default, the natural order is used")
+    @SystemProperty(value = "Enables fields to be written by BinaryMarshaller in sorted order. " +
+        "By default, the natural order is used",
```

Comment:
```suggestion
    @SystemProperty(value = "Enables fields to be written by BinaryMarshaller in sorted order. Use natural order in case the property is disabled " +
        "By default, the sorted order is used",
```